nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -143,16 +150,19 @@ public void beforeEach() throws Exception {
         // add fake connect request handler for package downloads
         server.setHandler(new FakeConnectDownloadHandler());
 
-        Environment environment = Environment.getDefault();
+        environment = Environment.getDefault();
         environment.setProperty(Environment.DISTRIBUTION_NAME, "server");
         environment.setProperty(Environment.DISTRIBUTION_VERSION, "8.3");
+        environment.setProperty(ConnectBroker.LAUNCHER_CHANGED_PROPERTY, "false");
+
         connectBroker = new ConnectBroker(environment);
+        connectBroker.setPendingFile(environment.getData().toPath().resolve("installAfterRestart.log"));
```

Comment:
use `org.nuxeo.launcher.config.ConfigurationGenerator.getInstallFile()`

Reply:
I'd like to. Unfortunately, `TestConnectBroker` does not have `ConfigurationGenerator` in its dependencies.